Zhuk.University.Tachka.Web/Pages/Carlist.cshtml

```diff
@@ -0,0 +1,81 @@
+﻿@page
+@model CarlistModel
+
+@{
+    ViewData["Title"] = "Car List";
+}
+
+<table class="table">
+    <thead>
+        <tr>
+            <th>
+                @Html.DisplayNameFor(model => model.Cars[0].Id)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Cars[0].Name)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Cars[0].Model)
+            </th>
+            <th>
+                @Html.DisplayNameFor(model => model.Cars[0].Price)
+            </th>
+        </tr>
+    </thead>
```

Comment:
```suggestion
    <thead>
      <tr>
          @foreach (var property in typeof(Zhuk.University.Tachka.Models.Database.Car).GetProperties())
          {
              <th>@Html.DisplayNameFor(model => model.Cars[0].GetType().GetProperty(property.Name))</th>
          }
      </tr>
    </thead>
```

Reply:
By making this change, the table headers will be dynamically generated based on the properties of the Car model, allowing for flexibility and easier maintenance when the model changes.

Reply:
It's table, only for test's. In future i will use other feature.